### PR TITLE
feature/FOUR-14541: When a new +MAIL DRIVER is deleted, this is not redirected to Email default Settigns tab

### PR DESCRIPTION
**Pull request by luNunezProcessmaker**
## Issue & Reproduction Steps
After delete the MAIL DRIVER the page is not redirect to Email default Settings 

## How Test
Go to Settings
Create a new Email 
Delete the new email

## Related Tickets & Packages
- https://processmaker.atlassian.net/browse/FOUR-14541

## Code Review Checklist
- [ ] I have pulled this code locally and tested it on my instance, along with any associated packages.
- [ ] This code adheres to [ProcessMaker Coding Guidelines](https://github.com/ProcessMaker/processmaker/wiki/Coding-Guidelines).
- [ ] This code includes a unit test or an E2E test that tests its functionality, or is covered by an existing test.
- [ ] This solution fixes the bug reported in the original ticket.
- [ ] This solution does not alter the expected output of a component in a way that would break existing Processes.
- [ ] This solution does not implement any breaking changes that would invalidate documentation or cause existing Processes to fail.
- [ ] This solution has been tested with enterprise packages that rely on its functionality and does not introduce bugs in those packages.
- [ ] This code does not duplicate functionality that already exists in the framework or in ProcessMaker.
- [ ] This ticket conforms to the PRD associated with this part of ProcessMaker.

ci:deploy
ci:connector-send-email:feature/FOUR-14541
ci:next

### CODE REVIEW

**Comment by tokensPM** [comment]
QA server K8S was successfully deployed https://ci-ee38f84240.engk8s.processmaker.net

File: resources/js/admin/settings/components/SettingsMenuCollapse.vue [hunk 1]
```diff
@@ -79,11 +80,10 @@ export default {
         .then((response) => {
           this.transformResponse(response.data.data);
           this.checkCollapedMenus();
-          if (this.firstTime) {
-            this.selectFirstItem();
-          }
           if (this.changeEmailServers) {
             this.checkChangeEmailServer();
+          } else if (this.firstTime) {
```
Comment (fagubla) [inline resources/js/admin/settings/components/SettingsMenuCollapse.vue:85]: remove else

**Review by fagubla — CHANGES_REQUESTED** [review]

**Comment by tokensPM** [comment]
QA server K8S was successfully deployed https://ci-ee38f84240.engk8s.processmaker.net

**Review by fagubla — APPROVED** [review]

**Comment by tokensPM** [comment]
QA server K8S was successfully deployed https://ci-ee38f84240.engk8s.processmaker.net

**Pull request merged**